packages/composites/bitcoin-json-rpc/src/endpoint/getblockchaininfo.ts

```diff
@@ -5,7 +5,7 @@ import { Validator, Requester } from '@chainlink/ea-bootstrap'
 export const NAME = 'getblockchaininfo'
 const DEFAULT_FIELD = 'difficulty'
 
-export const supportedEndpoints = [NAME, 'difficulty']
+export const supportedEndpoints = [NAME, 'difficulty', 'height']
```

Comment:
Should we be adding a convention like this here?

https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/sources/coingecko/src/endpoint/global.ts#L6-L9

Reply:
indeed. we won't need to use switch statement in this case